### ISSUE TITLE
compute-image-packages has services that cannot be started or stopped due to missing binaries

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/distribution#1595]
The following services provides by the compute-image-packages-services subpackage cannot be started and stopped:
```
# /usr/lib/systemd/system/google-ip-forwarding-daemon.service cannot be started/stopped because /usr/bin/google_ip_forwarding_daemon does not exist
# /usr/lib/systemd/system/google-network-setup.service cannot be started/stopped because /usr/bin/google_network_setup does not exist
```

**Comment by phmccarty** [comment]
Those two binaries were removed from `compute-image-packages` a while ago, but the associated service files were not.

I am updating this package to the latest version and refreshing the service files that are installed/enabled.

**Comment by phmccarty** [comment]
Will be fixed with `compute-image-packages-20191210-31`.

**Comment by phmccarty** [comment]
Fixed in release `32020`.
